app/forms/user_onboarding_form.rb

```diff
@@ -0,0 +1,14 @@
+class UserOnboardingForm
+  include ActiveModel::Model
+
+  attr_accessor :user, :accepted_privacy_policy_at, :accepted_terms_at
+
+  validates :terms_of_service, acceptance: true
+  validates :privacy_policy, acceptance: true
+
+  def save
+    return unless valid?
+
+    user.update!(accepted_privacy_policy_at: Time.current, accepted_terms_at: Time.current)
+  end
+end
```

Comment:
What's the advantage of using this form to updating the user directly? I can't see where it's adding extra value.

Reply:
There's a validation to make sure that the users checked the checkbox. We can remove that if you don't want that.